src/expr/scalar/func.rs

```diff
@@ -639,6 +639,26 @@ fn round_decimal<'a>(a: Datum<'a>, b: Datum<'a>, a_scale: u8) -> Datum<'a> {
     Datum::from(decimal.round(round_to).significand())
 }
 
+fn convert_from<'a>(a: Datum<'a>, b: Datum<'a>) -> Result<Datum<'a>, EvalError> {
+    let encoding_name = b.unwrap_str().to_lowercase();
+    if encoding_name != "utf8" {
```

Comment:
Limiting to utf8 for now is fine, but PostgreSQL supports multiple ways of spelling the encoding, like "UTF8", "utf-8", and "UtF&&&-8" (seriously). Skipping all that for now is fine, but we should at least be internally consistent about what spellings we support. The `length` function currently does some encoding name parsing that we should adopt.